src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -173,51 +203,42 @@ public void Start()
                     sqlCommand.CommandText = @"
                         SET XACT_ABORT ON
                         BEGIN TRANSACTION
-
                         INSERT INTO dbo.ResourceType (Name) 
                         SELECT value FROM string_split(@resourceTypes, ',')
                         EXCEPT SELECT Name FROM dbo.ResourceType WITH (TABLOCKX); 
-
                         -- result set 1
                         SELECT ResourceTypeId, Name FROM dbo.ResourceType;
-
                         INSERT INTO dbo.SearchParam (Uri)
                         SELECT * FROM  OPENJSON (@searchParams) 
                         WITH (Uri varchar(128) '$.Uri')
-                        EXCEPT SELECT Uri FROM dbo.SearchParam;
```

Comment:
Curious, was there a reson for removing the semicolon?

Reply:
Nope, thanks for catching this. Added it back in in c1d9233.